test/hex_web/api/router_test.exs

```diff
@@ -38,6 +39,19 @@ defmodule HexWeb.API.RouterTest do
     :ok
   end
 
+  @doc """
+  Creates and returns an auth key for a given user
+  """
+  @spec test_key_for(String.t | User.t) :: String.t
+  def test_key_for(username) when is_binary(username) do
```

Comment:
I assume it meant to be test helper and used only in `router_test.exs`, considering that it should rather be `defp` with `@doc` removed.